### DISCUSSION TITLE
怎么让Github加载的快一点

### DISCUSSION THREAD

**Discussion by YuTinMIn** [share121/inter-knot#2432] [其他]
![啊？](https://github.com/user-attachments/assets/a98ef199-104a-4d5d-8a5b-c395ea1ae81a)
加载的太慢了，我点一下要加载2秒（可能已经比较快了），有没有大佬给出一个免费又好用的加速网站的方案？

**Comment by hewanghai123** [comment]
电脑的话可以用Watt Toolkit

**Reply by YuTinMIn** [reply]
我就是用Watt Toolkit的，但有点卡

**Comment by shuiyuan2023** [comment]
可以用steamcommunity_302，挺好用的

**Reply by YuTinMIn** [reply]
谢谢推荐，确实好用

**Comment by HifumiMizuhara** [comment]
用小猫，详细我不多说，咳咳

**Comment by Qweulin** [comment]
想要体验好的话还是买个机场订阅罢（

**Comment by did1181** [comment]
小黑盒加速器好像支持github,但我是用梯子（笑）

**Comment by ShinyuanJ** [comment]
最好是自己买个比较好的节点放v2或者clash上。不过好点的基本是包年一年二三百，找人拼一个人也要一百多，看你能否负担得了

**Comment by edge-stroke** [comment]
整个梯子罢

**Comment by Tuskiy** [comment]
萌云。一个月10块100G，不下东西还是绰绰有余的<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/94252e623d6ce6c3a2463be2afc23751_1000866240408823916.gif" alt="爱莉希雅-扭扭" width="75" >

**Comment by isle200inf** [comment]
**机场，便宜，怕跑路可以一个月一个月买**

**Comment by snivm** [comment]
自用，稳定，按流量计费，不限时长，流量用完前不会重置，每月不会刷新，支持Netflix和chatgpt,最低1元套餐，适合轻量使用，性价比高，mojie.me/#/register?code=XdLgjGwe​

**Comment by xingyu42** [comment]
肉身到香港我打包票绝对快到吓人 (-^〇^-)

**Reply by fzdqz** [reply]
都不用去香港，来广州就已经可以流畅刷绳网了

**Comment by fzdqz** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/b171cc5b6c8d8b6257a3bf8163fb2680_567796524796359289.png" alt="阿姬得意" width="75" >来沿海城市，都可以不用魔法流畅刷绳网的
